sandbox.sh

```diff
@@ -121,8 +121,7 @@ trusted_validator_membership_change_json() {
     i=$(echo "$VALIDATOR" | awk -F';' '{ print $1 }')
     ADDRESS=$(echo "$VALIDATOR" | awk -F';' '{ print $4 }')
     if [ "$i" != 0 ]; then
-      printf ",
-"
+      printf ","
```

Comment:
Why? We should probably use jq to pretty print this part instead of printf, but I'm not sure why you change this.

Reply:
I just move the `"` up next to the `printf ",`.